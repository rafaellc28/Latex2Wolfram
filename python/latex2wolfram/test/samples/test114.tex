% Test nulity matrix
nullity(\begin{bmatrix}
  6 & -11 & 13\\
  4 &  -1 &  3\\
  3 &   4 & -2\\
\end{bmatrix})
