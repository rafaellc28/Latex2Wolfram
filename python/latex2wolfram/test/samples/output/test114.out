nullity({{6, -11, 13}, {4, -1, 3}, {3, 4, -2}})
